.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+// Coloque su configuración en este archivo para sobrescribir la configuración predeterminada y de usuario.
```

Comment:
The risk here is to have users using different settings, but we can limit the impact using tslinter rules. I was thinking about keeping some major editors like atom and vscode.